packages/ui/lib/components/Detail/Detail.tsx

```diff
@@ -0,0 +1,101 @@
+import { Tooltip } from '../Tooltip/Tooltip'
+import { IconType } from 'react-icons'
+import { Icon } from '../Icon/Icon'
+import { Size, SizeStyleProp } from '~/types'
+import { Placeholder } from '../Placeholder'
+import { classed } from '@tw-classed/react'
+
+export interface DetailProps {
+  label: React.ReactNode
+  icon?: IconType
+  iconSize?: number
+  value?: React.ReactNode
+  loading?: boolean
+  inline?: boolean
+  labelSize?: Size
```

Comment:
Size it according to the entire component when possible. I think label can be like that.

Reply:
i added `labelSize` and `valueSize` to make it more flexible as possible, for example in lock card we have this: 

```
- label: tiny
- value: large 
```

**Example**

<img width="148" alt="Screenshot 2023-02-22 at 15 30 47" src="https://user-images.githubusercontent.com/20865711/220671105-b5fba721-512a-4330-98c6-ca4730a848e8.png">